### PR TITLE
fix: override type in CLI to allow wrapping contract calls

### PR DESCRIPTION
**Pull request by pcarranzav**

### CODE REVIEW

**Review by Maikol — DISMISSED** [review]

**Review by tmigone — CHANGES_REQUESTED** [review]
Nice! 

Can you also make this change in the SDK? `packages/sdk/src/deployments/lib/contracts/tx-log.ts`

**Comment by tmigone** [comment]
Btw this code duplication problem should be resolved once we get rid of the cli folder.

File: packages/sdk/src/deployments/lib/contracts/tx-log.ts [hunk 1]
```diff
@@ -66,7 +71,7 @@ export function wrapCalls(contract: Contract, contractName: string): Contract {
     wrappedContract.functions[fn] = override
```
Comment (tmigone) [inline packages/sdk/src/deployments/lib/contracts/tx-log.ts:71]: We should remove:
```
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```

**Review by tmigone — APPROVED** [review]
Approving so you can merge but left one comment!

**Review by tmigone — APPROVED** [review]

**Pull request merged**